prometheus-exporters/octobus-query-exporter/vendor/octobus-query-exporter/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.0.20
+version: 1.0.21
```

Comment:
remove this change and do a proper chart bump for prometheus-vmware-rules `Chart.yaml`.